app/jobs/repository_item_date_reminder_job.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class RepositoryItemDateReminderJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    RepositoryDateTimeValue
+      .where(notification_sent: false)
+      .where('data <= ?', DateTime.current)
+      .includes(repository_cell: { repository_column: :repository_snapshot })
+      .find_each do |date_time_value|
+      next if date_time_value.repository_cell.repository_column.repository_snapshot.present?
```

Comment:
Here we need to just skip items from snapshots, not skip items that have snapshots. So we just need to join the repository and filter it by 'type'.